### ISSUE TITLE
Flutter OneSignal Notification Sound Not Working

### ISSUE THREAD

**Issue by karatasemre** [OneSignal/OneSignal-Flutter-SDK#218]
I use OneSignal to send notifications in my application I developed with Flutter. Notifications are coming. I also attached the audio file in accordance with the instructions on the site. But when the notification comes, the voice I added does not come. The phone's default notification sound is playing. Anyone have an idea about this?

res/raw/onesignal_default_sound.wav

res/xml/keep.xml

kepp.xml

`<?xml version="1.0" encoding="utf-8"?>
<resources
    xmlns:tools="http://schemas.android.com/tools"
    tools:keep="@raw/onesignal_default_sound"/>`

**Comment by jkasten2** [comment]
@karatasemre Sorry for the confusion on `onesignal_default_sound`. It is deprecated as it does not work with Android 8+ devices. Instead you must used Notification Categories noted here
https://documentation.onesignal.com/docs/customize-notification-sounds#section-3-send-notifications-with-sounds

We have removed `onesignal_default_sound` from the [Can I set a default sound?](https://documentation.onesignal.com/docs/customize-notification-sounds#section-can-i-set-a-default-sound) section.

In summary to have a default sound you will need to create a [Notification Category on the OneSignal Dashboard](https://documentation.onesignal.com/docs/android-notification-categories) then create a [OneSignal Notification Template](https://documentation.onesignal.com/docs/templates) using your Notification Category. Lastly make sure to use the template on all future notifications.

**Comment by irshadkhaleelp** [comment]
This same issue persist me too. this is happening on the cases of after taking apk, notifications are coming not the custom sound. but working when the device is on debug (working time), for both emulators and physical devices
